docs/operations/api-reference.md

```diff
@@ -102,6 +102,15 @@ Returns the number of segments left to load until segments that should be loaded
 
 Returns the number of segments left to load in each tier until segments that should be loaded in the cluster are all available. This includes segment replication counts.
 
+* `/druid/coordinator/v1/loadstatus?full?computeUsingClusterView`
+
+Returns the number of segments left to load in each tier until segments that should be loaded in the cluster are all available.
+This includes segment replication counts. When computing the number of segments left to load, the number of nodes
+available in the cluster that are of a service type that can load the segment is considered. A segment is considered fully
+loaded if it is replicated as many times as configured in the corresponding load rule, or if there are as many replicas of
```

Comment:
```suggestion
- Druid had replicated it the number of times configured in the corresponding load rule.
- Or the number of replicas for the segment in each tier where it configured to be replicated equals the available nodes of a service type that are currently allowed to load the segment in the tier.
```

Reply:
Suggest "equals the number of available nodes" instead in the suggested revision